include/rocksdb/metadata.h

```diff
@@ -153,6 +156,44 @@ struct LiveFileMetaData : SstFileMetaData {
   LiveFileMetaData() : column_family_name(), level(0) {}
 };
 
+// The MetaData that describes a Blob file
```

Comment:
Could you also add the new API we discussed that would expose a map of column family name -> `ColumnFamilyMetaData`?